IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcCourse/Documentation.md

```diff
@@ -4,3 +4,27 @@ Structurally a course does not have capacity to carry loads over open span, or t
 * Graded sand layers
 * Cement bounded material (CBM)
 * Asphalt layers
+
+>NOTE For road applications, the following types of courses were identified but not added to the schema model due to the fact that the elements (including terms and definitions) can vary greatly between nations; instead, these would be typical values of the inherited attribute ObjectType:
+
+- **Paving** : Type of Course directly in contact with traffic loads e.g. in gravel road (also Surface layer or Wearing course).
+- **AnticapillaryLayer** : Type of Course, layer preventing capillary rise of water from underlying layers. 
+>NOTE Definition according to PIARC: Protective bottom layer with no fines, designed to prevent capillary rise of water from the subgrade soil or from underlying layers.
+- **AntifreezingLayer** : Type of Course, layer intended to prevent frost from penetrating into the subgrade. Also: Frost blanket course.
+- **BaseCourse** : A layer immediately beneath the surface of binder course providing additional load distribution and contributing to the sub-surface drainage.
+- **BinderCourse** : A layer to distribute loads to the base course.
+- **CappingLayer** : Top layer of subgrade to improve its load bearing capacity. Definition from PIARC: Optional layer of granular or treated material on top of the subgrade and immediately below formation level, to provide improved foundation for the subbase layer [CEN].
+- **DrainingCourse** : Layer of pervious material to relieve pore pressure or facilitate drainage of overlying layers.
+- **TackCoat** : A thin coating of tar or asphalt applied before a road is laid to form an adhesive bond.
+- **LayingCourse** : Type of Course, layer of material providing a bed for which block (stone) paving.
+- **RegulatingCourse** : Type of Course, layer of variable thickness for adjusting a surface so meet specified even level. Also: Levelling course. 
+>NOTE Definition according to PIARC: Course of variable thickness applied to an existing course or surface to provide the necessary profile for a further course of constant thickness (CEN).
+- **Sealing** : A layer of impermeable material.
+- **SubBaseCourse** : A layer between subgrade and base course.
```

Comment:
*BaseCourse* exists in the list, *SubgradeCourse* doesn't. Intentional?

Reply:
If you with subgrade course mean subgrade, then I believe this is part of IfcEarthworksFill. See https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/ifcearthworksfilltypeenum.htm